k8s-tester/tester.go

```diff
@@ -233,6 +234,13 @@ func (ts *tester) createTesters() {
 		ts.cfg.AddOnStressInCluster.Client = ts.cli
 		ts.testers = append(ts.testers, stress_in_cluster.New(ts.cfg.AddOnStressInCluster))
 	}
+	if ts.cfg.AddOnFalco != nil && ts.cfg.AddOnFalco.Enable {
```

Comment:
Can this be located above? Like after csi-ebs? I would like stress tests to be at the end :)

Reply:
How about after `AddOnKubernetesDashboard`, we run Falco?